### PR TITLE
[xdoctest] reformat example code with google style in 271

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others 
### PR changes

Others 
### Description

https://github.com/PaddlePaddle/Paddle/issues/55629

### CODE REVIEW

File: python/paddle/utils/dlpack.py [hunk 1]
```diff
@@ -39,13 +39,14 @@ def to_dlpack(x):
...
-            print(dlpack)
-            # <capsule object "dltensor" at 0x7f6103c681b0>
+            >>> import paddle
+            >>> # x is a tensor with shape [2, 4]
+            >>> x = paddle.to_tensor([[0.2, 0.3, 0.5, 0.9],
+            ...                       [0.1, 0.2, 0.6, 0.7]])
+            >>> dlpack = paddle.utils.dlpack.to_dlpack(x)
+            >>> print(dlpack)
+            >>> # the address will change in every run
+            >>> # <capsule object "dltensor" at 0x7f6103c681b0>
```
Comment (SigureMo) [inline python/paddle/utils/dlpack.py]: 放在输出，`the address will change in every run` 作为 SKIP 的理由

Comment (Liyulingyue) [inline python/paddle/utils/dlpack.py]:
```suggestion
            <capsule object "dltensor" at 0x7f6103c681b0>
```

File: python/paddle/utils/dlpack.py [hunk 2]
```diff
@@ -39,13 +39,14 @@ def to_dlpack(x):
...
         .. code-block:: python
 
-            import paddle
-            # x is a tensor with shape [2, 4]
-            x = paddle.to_tensor([[0.2, 0.3, 0.5, 0.9],
-                                  [0.1, 0.2, 0.6, 0.7]])
-            dlpack = paddle.utils.dlpack.to_dlpack(x)
-            print(dlpack)
-            # <capsule object "dltensor" at 0x7f6103c681b0>
+            >>> import paddle
```
Comment (Liyulingyue) [inline python/paddle/utils/dlpack.py:42]:
```suggestion
            >>> # doctest: +SKIP('the address will change in every run')
            >>> import paddle
```

File: python/paddle/utils/dlpack.py [hunk 3]
```diff
@@ -39,13 +39,14 @@ def to_dlpack(x):
...
+            >>> # x is a tensor with shape [2, 4]
+            >>> x = paddle.to_tensor([[0.2, 0.3, 0.5, 0.9],
+            ...                       [0.1, 0.2, 0.6, 0.7]])
+            >>> dlpack = paddle.utils.dlpack.to_dlpack(x)
+            >>> print(dlpack)
+            >>> # the address will change in every run
```
Comment (Liyulingyue) [inline python/paddle/utils/dlpack.py]:
```suggestion
```

File: python/paddle/utils/dlpack.py (hunk 1 above)
Comment (Liyulingyue) [reply python/paddle/utils/dlpack.py]: done

File: python/paddle/utils/dlpack.py [hunk 4]
```diff
@@ -39,13 +39,14 @@ def to_dlpack(x):
...
         .. code-block:: python
 
-            import paddle
-            # x is a tensor with shape [2, 4]
-            x = paddle.to_tensor([[0.2, 0.3, 0.5, 0.9],
-                                  [0.1, 0.2, 0.6, 0.7]])
-            dlpack = paddle.utils.dlpack.to_dlpack(x)
-            print(dlpack)
-            # <capsule object "dltensor" at 0x7f6103c681b0>
+            >>> # doctest: +SKIP('the address will change in every run')
```
Comment (SigureMo) [inline python/paddle/utils/dlpack.py]: 只跳过输出对比，而不是整段跳过

File: python/paddle/utils/dlpack.py [hunk 5]
```diff
@@ -39,13 +39,14 @@ def to_dlpack(x):
...
+            >>> import paddle
+            >>> # x is a tensor with shape [2, 4]
+            >>> x = paddle.to_tensor([[0.2, 0.3, 0.5, 0.9],
+            ...                       [0.1, 0.2, 0.6, 0.7]])
+            >>> dlpack = paddle.utils.dlpack.to_dlpack(x)
+            >>> print(dlpack)
+            <capsule object "dltensor" at 0x7f6103c681b0>
```
Comment (Liyulingyue) [inline python/paddle/utils/dlpack.py:49]:
```suggestion
            >>> # doctest: +SKIP('the address will change in every run')
            <capsule object "dltensor" at 0x7f6103c681b0>
```

File: python/paddle/utils/dlpack.py (hunk 4 above)
Comment (Liyulingyue) [inline python/paddle/utils/dlpack.py]:
```suggestion
```

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾

**Pull request merged**